modules/prompt/functions/prompt-pwd

```diff
@@ -12,13 +12,15 @@ setopt localoptions extendedglob
 local current_pwd="${PWD/#$HOME/~}"
 local ret_directory
 
-if [[ "$current_pwd" == (#m)[/~] ]]; then
+if zstyle -m ':prezto:module:prompt' pwd-length 'full'; then
+  ret_directory=${PWD}
+elif [[ "$current_pwd" == (#m)[/~] ]]; then
```

Comment:
Shouldn't this be the last branch before the `else` to avoid using this for other explicit options as well?